test/bedrock_Azure_mc_test.go

```diff
@@ -34,12 +36,44 @@ func addIPtoYAML(input string, ipaddress string) {
 func TestIT_Bedrock_AzureMC_Test(t *testing.T) {
 	t.Parallel()
 
-	//Generate a random cluster name to prevent a naming conflict and map variables to tfvars
+	// Generate a common infra resources for integration use with azure multicluster environment
 	uniqueID := strings.ToLower(random.UniqueId())
 	k8sName := fmt.Sprintf("gtestk8s-%s", uniqueID)
-	tmName := k8sName + "-tm"
 
+	location := os.Getenv("DATACENTER_LOCATION")
```

Comment:
Build is failing because we enabled the backend for the tfstatekey to be stored in an azure storage account. Check out [L22-L44](https://github.com/Microsoft/bedrock/blob/master/test/bedrock_Azure_common_kv_test.go#L28-#L44) which adds the Backend credentials from the env and separates the terraform init and apply.